src/Config.ts

```diff
@@ -321,9 +321,13 @@ export interface ConfigParams {
    */
   precisionEpsilon: number,
   /**
-   * Sets calculations' precision level.
+   * Sets the precision level of calculations' output.
    *
-   * Numerical outputs are rounded to the `precisionRounding` number of digits after the decimal.
+   * Internally, all arithmetic operations are performed using JavaScript's built-in numbers.
+   * But when HyperFormula exports a cell's value, it rounds the output
+   * to the `precisionRounding` number of significant digits.
+   *
+   * We recommend setting `precisionRounding` to a value between `1` and `14`.
```

Comment:
```suggestion
   * For a typical use we recommend setting `precisionRounding` to a value between `10` and `14`.
   * Too small value might result in losing precision of large numbers, e.g. 100005 becomes 100010 when `precisionRounding=4`.
```

Reply:
Please decide if we want to include the second line with the example.

Reply:
@bardek8 - sure, added: 8ee4f1c